probe_test.go

```diff
@@ -626,3 +626,49 @@ func TestLicenseStatus(t *testing.T) {
 		t.Fatalf("metric compare: err %v", err)
 	}
 }
+
+func TestLinkStatus(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/system/link-monitor", "testdata/link-monitor.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeLinkMonitor, c, r) {
+		t.Errorf("probeLinkMonitor() returned non-success")
+	}
+
+	em := `
+# HELP fortigate_link_bandwidth_rx Bandwidth available on this link for sending
+# TYPE fortigate_link_bandwidth_rx gauge
+fortigate_link_bandwidth_rx{link="wan1",link_group="wan-mon",vdom="root"} 33557
+# HELP fortigate_link_bandwidth_tx Bandwidth available on this link for sending
+# TYPE fortigate_link_bandwidth_tx gauge
+fortigate_link_bandwidth_tx{link="wan1",link_group="wan-mon",vdom="root"} 68657
+# HELP fortigate_link_jitter electronical jitter on this link
+# TYPE fortigate_link_jitter gauge
+fortigate_link_jitter{link="wan1",link_group="wan-mon",vdom="root"} 1.1268666982650757
```

Comment:
Unit is missing here as well, not clear what this is - hopefully the API defines it so it is not totally unknown?

Nit: Capital letter on the first letter of the description

Reply:
see comment above